srcpkgs/onlyoffice/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'onlyoffice'
+pkgname=onlyoffice
+version=7.0.1
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz"
+depends="libstdc++ curl xdg-utils dejavu-fonts-ttf GConf gtk+3 liberation-fonts-ttf libpulseaudio gstreamer1 nss alsa-lib gst-plugins-base1 gst-plugins-ugly1 nspr hicolor-icon-theme"
+short_desc="Office suite that combines text, spreadsheet and presentation editors"
+maintainer="Toromino <foxhkron@cybre.club>"
+license="AGPL-3.0-only"
+homepage="https://www.onlyoffice.com"
+distfiles="https://github.com/ONLYOFFICE/DesktopEditors/releases/download/v${version}/onlyoffice-desktopeditors_amd64.deb"
```

Comment:
Yep.

Not accepted. We don't need more big binary blobs we just download. We prefer to build these things ourself.